cpp/src/backends/torch_scripted/handler/base_handler.cc

```diff
@@ -166,7 +166,6 @@ void BaseHandler::Postprocess(
   for (const auto& kv : idx_to_req_id) {
     try {
       auto response = (*response_batch)[kv.second];
-      auto msg = torch::argmax(data[kv.first]).to(torch::kFloat32);
       response->SetResponse(200, "data_type",
                             torchserve::PayloadType::kDATA_TYPE_BYTES,
                             torch::pickle_save(data[kv.first]));
```

Comment:
why aren't we using softmax here?

Reply:
This is basehandler which can be used by any model, not image classify handler.